setup.py

```diff
@@ -38,6 +38,7 @@ def package_files(package_dir, subdirectory):
     'version': version.version,
     'package_data': {'hera_qm': data_files},
     #    'install_requires': ['numpy>=1.10', 'scipy', 'pyuvdata', 'astropy>1.2', 'aipy']
```

Comment:
I'll leave that to @adampbeardsley b/c I haven't been editing the setup.py script